client/post-editor/editor-drawer/copy-post.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import PostSelector from 'my-sites/post-selector';
+
+class EditorDrawerCopyPost extends Component {
+
+	static propTypes = {
+		site: PropTypes.object.isRequired,
+		post: PropTypes.object.isRequired,
+		translate: PropTypes.func,
+	};
+
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			selectedPostID: '',
```

Comment:
I noticed that `null` gave me an error, but it doesn't anymore.
I'm wondering if something changed overnight in my Vagrant!